pkg/engine/mutation.go

```diff
@@ -39,7 +48,7 @@ func Mutate(policyContext PolicyContext) (resp response.EngineResponse) {
 
 	for _, rule := range policy.Spec.Rules {
 		//TODO: to be checked before calling the resources as well
-		if !rule.HasMutate() {
+		if !rule.HasMutate() && !strings.Contains(PodControllers, resource.GetKind()) {
```

Comment:
Use slice of strings instead of comma-separated string of values.

Reply:
Any reason to change to a slice?